<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPropertiesDialog</class>
 <widget class="QDialog" name="ProjectPropertiesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="ProjectPropertiesTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="BuildConfigurationTab">
      <attribute name="title">
       <string>Build Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <item>
        <widget class="QGroupBox" name="simulatorGroupBox">
         <property name="title">
          <string>Simulator Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="simulatorCompilerValueLayout">
            <item>
             <widget class="QComboBox" name="simulatorCompilerComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="simulatorCompilerLabel">
            <property name="text">
             <string>Compiler</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="targetGroupBox">
         <property name="title">
          <string>Target Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="targetCompilerLabel">
            <property name="text">
             <string>Compiler</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="targetCompilerValueLayout">
            <item>
             <widget class="QComboBox" name="targetCompilerComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="endiannessLabel">
            <property name="text">
             <string>Endianess</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="targetEndianessComboBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CANSpecificationTab">
      <attribute name="title">
       <string>CAN Specification</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>441</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="canVerticalLayout">
        <item>
         <layout class="QHBoxLayout" name="canHorizontalLayout">
          <item>
           <widget class="QLineEdit" name="canPathLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="changeCanSpecButton">
            <property name="text">
             <string>Change/Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="canChangeSummaryTextBrowser"/>
        </item>
        <item>
         <widget class="QPushButton" name="canCopyToClipboardButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Copy to clipboard</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
